### ISSUE TITLE
Minor Issue - Quantity does not seem to be working

### ISSUE THREAD

**Issue by fmegahed** [jnolis/ggirl#15]
Hi @jnolis ,

Thank you for your excellent package. I tried to make a couple of prints from one of our Maps; however, when I went to the app site, there was only one plot ($30 instead of $60). I am not sure if this is a bug or something wrong with my code. 

For your reference, I am attaching the code that I used below (not really a minimal example but not huge either): 

if(require(remotes) == FALSE) install.packages("remotes")
if(require(pacman)==FALSE) install.packages("pacman")
if(require(ggirl) == FALSE) remotes::install_github("jnolis/ggirl")

```
pacman::p_load(tidyverse, magrittr, lubridate, albersusa, tigris) # loading the required packages


clusterCounties = readRDS(url("https://github.com/fmegahed/covid19/blob/master/Data/clusterCounties.rds?raw=true")) # myData
clusterCounties$fips = str_pad(clusterCounties$key_numeric, width = 5, side = 'left', pad = '0')
clusterCounties %<>% ungroup()


cty_sf = counties_sf("aeqd") %>% filter(!state %in% c('Alaska', 'Hawaii')) # map from albersusa package
cty_sf %<>% left_join(clusterCounties[, c('fips', 'cluster_group')], by = 'fips') # adding cluster_group to cty_sf


cty_sf %>% # ggplot for the map inspired by https://stackoverflow.com/a/60843261/10156153
  ggplot() +
  geom_sf(mapping = aes(fill = cluster_group), color = "black") +
  coord_sf(datum = NA) +   
  scale_fill_brewer(palette = 'Set2') +
  theme_bw(base_size = 24) + theme(legend.position="bottom", panel.border = element_blank()) +
  labs(caption = 'Based on Data from March 01, 2020 - January 02, 2021') -> p


delivery_address <- address(name = "My Name", address_line_1 = "My Address",
                            address_line_2 = "", city = "Oxford", state = "OH",
                            postal_code = "45056", country = "US")
contact_email = "myEmail@gmail.com"

ggartprint(p, background = "#FEFDF7", size="11x14", orientation = "landscape", quantity = 2,
           contact_email = contact_email, address = delivery_address)
```

**Comment by jnolis** [comment]
Hi! I see this error and can reproduce it. I will fix it soon, although for now an easy workaround would be to make two separate orders (which is functionally the same). Thanks!

**Comment by jnolis** [comment]
This was fixed in 2.0.1 (on the server side)
